docs/getting-started-guides/docker-multinode.md

```diff
@@ -53,7 +49,9 @@ Please install Docker 1.6.2 or Docker 1.7.1.
 
 ## Prerequisites
 
-1. You need a machine with docker of right version installed.
+The only thing you need is a machine with **Docker 1.8.x or higher**
+
+It might work with older docker versions, but that isn't supported.
```

Comment:
AFAIU the bug referenced here affects users only if they run docker daemon differently. We don't do this in our guide. I'd either just delete this requirement at all, or move it to a separate section `Troubleshooting`. WDYT?